src/pages/GiftAidForm/defaultGiftaidFields.json

```diff
@@ -35,7 +35,7 @@
     "name": "lastname",
     "label": "Last name",
     "required": true,
-    "pattern": "^[A-Za-z0-9]+[ \\- ,.()\\/&\\'\\w]+$",
+    "pattern": "^[a-zA-Z0-9'.,/()&-][a-zA-Z0-9'.,/()& -]{0,24}$",
```

Comment:
Same here, means the last name can begin with `'.,/()&-`
Also, `'`, `/` and `-` must be escaped with `\` to work